apps/amiapp_flutter/android/app/google-services.json

```diff
@@ -5,6 +5,42 @@
     "storage_bucket": "remote-habits.appspot.com"
   },
   "client": [
+    {
```

Comment:
File got updated while going through the flutter-fire documentation. The docs ask you to run `flutterfire configure` and it automatically downloads updates to the google services files. 

I decided to commit these files to be extra sure that installing flutter-fire works as `flutterfire configure` is what made this change for us.